test1-prod.js

```diff
@@ -7,4 +7,6 @@ if(fooProd  === "bar") {
   console.log( 'sha from event' )
   
   console.log( 'sha checking event name for real with logs' )
+  
+  console.log( 'this line is flagged by the CLI' )
```

Comment:
Codacy has a fix for the issue: [Replace `·'this·line·is·flagged·by·the·CLI'·)` with `"this·line·is·flagged·by·the·CLI");`](https://app.dev.codacy.org/gh/troubleshoot-codacy/eslint-test-prod/pullRequest?prid=33431)
```suggestion
  console.log("this line is flagged by the CLI");
```